apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -730,6 +726,13 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
                                     showPredictions={ false }
                                     customLabel={ allowedOriginsErrorLabel }
                                 />
+                                <Hint>
+                                    The HTTP origins that host your web application. You can define multiple web
```

Comment:
@yasinmiran Shouldn't these be coming from I18n file?

Reply:
Yes, it must refer from i18n/console. I will create an issue for this.